src/main/scala/com/yotpo/metorikku/test/KeyColumns.scala

```diff
@@ -0,0 +1,63 @@
+package com.yotpo.metorikku.test
+import org.apache.spark.sql.DataFrame
+
+import scala.collection.mutable
+
+object KeyColumns {
```

Comment:
Maybe this deserves a class? with a toString? couple of instantiation option etc.